root/area/AreaLayout.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+import type {Node as ReactNode} from 'react';
+
+import Layout from '../layout';
+import AreaSidebar from '../layout/components/sidebar/AreaSidebar';
+import {hyphenateTitle, l} from '../static/scripts/common/i18n';
+
+import AreaHeader from './AreaHeader';
+
+type Props = {|
+  +children: ReactNode,
+  +entity: AreaT,
+  +fullWidth?: boolean,
+  +page: string,
+  +title?: string,
+|};
+
+const AreaLayout = ({
+  children,
+  entity: area,
+  fullWidth,
+  page,
+  title,
+}: Props) => (
+  <Layout
+    title={title ? hyphenateTitle(l(area.name), title) : l(area.name)}
```

Comment:
The TT file calls out to `area.l_name`, which isn't as simple: https://github.com/metabrainz/musicbrainz-server/blob/8ce0b73c6e6378542e39f0fac84b4f4ce8ca03ca/lib/MusicBrainz/Server/Entity/Area.pm#L22-L30

I suppose we should add a `localizeAreaName` utility to implement that logic. There's an `AREA_TYPE_COUNTRY` constant already in root/static/scripts/common/constants.js. Note that it should also use `l_countries`.